Arriba/Arriba/Model/Query/SelectQuery.cs

```diff
@@ -484,6 +512,7 @@ public SelectResult Merge(SelectResult[] partitionResults)
                         for (int columnIndex = 0; columnIndex < mergedBlock.ColumnCount; ++columnIndex)
                         {
                             mergedBlock.SetValue(itemIndex, columnIndex, partitionResults[bestPartition].Values.GetValue(nextIndexPerPartition[bestPartition], columnIndex));
+                            mergedOrderbyBlock.SetValue(itemIndex, 0, partitionResults[bestPartition].OrderByValues.GetValue(nextIndexPerPartition[bestPartition], 0));
```

Comment:
Copying the mergedOrderByBlock values should happen after this loop,right? Is it duplicating the copy for the count of other columns you returned?